controllers/index.js

```diff
@@ -0,0 +1,24 @@
+const CoffeeForm = require("../models/CoffeeForm");
+const asyncCatcher = require("../utils/asyncCatcher");
+const CoffeeFormService = require("../services/CoffeeFormService");
+// eslint-disable-next-line no-unused-vars
+
+const CoffeeFormInstance = new CoffeeFormService(CoffeeForm);
+// eslint-disable-next-line no-unused-vars
+exports.submit = asyncCatcher(async (req, res, next) => {
+  const coffeeForm = {
+    from: "62a1cb520f4d52f56ec0aca3",
+    to: "62a13f789b2ff58829c6b587",
+    title: req.body.title,
+    content: req.body.content,
+    accepted: false,
+  };
+
+  try {
+    CoffeeFormInstance.RegisterCoffeeForm(coffeeForm);
+  } catch (err) {
+    console.log("err", err);
+  }
```

Comment:
```suggestion
await CoffeeFormInstance.RegisterCoffeeForm(coffeeForm);
```

`asyncCatcher` 가 중앙에서 exception 을 핸들링하고 있기 때문에 try catch 문은 독립적인 에러핸들링을 위한 경우가 아니라면 없애주셔도 잘 작동할겁니당

그리고 await 이 빠진것 같네요!